open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive1_2.java

```diff
@@ -1415,7 +1415,7 @@ private RelationshipDef getExternalIdLinkRelationship()
          * Set up end 1.
          */
         final String                     end1EntityType               = "Referenceable";
-        final String                     end1AttributeName            = "resource";
+        final String                     end1AttributeName            = "resources";
```

Comment:
Are we allowed to change the name of an attribute in a previous metadata type archive (1.2 in this file)?  Wouldn't this break backwards compatibility?

Reply:
No - because it is documentation - not used in the code ....